publons/PublonsPlugin.inc.php

```diff
@@ -207,10 +209,10 @@ function handleTemplateFetch($hookName, $args) {
 
             switch ($template) {
                 case 'reviewer/review/reviewCompleted.tpl':
-                    $templateMgr->register_outputfilter(array(&$this, 'completedSubmissionOutputFilter'));
+                    $templateMgr->registerFilter('output', array(&$this, 'completedSubmissionOutputFilter'));
```

Comment:
the Smarty's `TemplateManager` used to have `unregister_outputfilter` and `register_outputfilter` functions but not any more. Now they use `registerFilter` and `unregisterFilter` instead, the link of definition is below:
https://www.smarty.net/docs/en/api.register.filter.tpl
https://www.smarty.net/docs/en/api.unregister.filter.tpl